src/parallel/KDDecomposition.cpp

```diff
@@ -115,6 +115,22 @@ void KDDecomposition::readXML(XMLfileUnits& xmlconfig) {
 						  "VecTuner version)?: "
 					   << (_heterogeneousSystems ? "yes" : "no") << endl;
 
+	std::string deviationReductionOperation;
+	xmlconfig.getNodeValue("deviationReductionOperation", deviationReductionOperation);
+	if (not deviationReductionOperation.empty()) {
+		if (deviationReductionOperation == "sum") {
+			_deviationReductionOperation = MPI_SUM;
+		} else if (deviationReductionOperation == "max") {
+			_deviationReductionOperation = MPI_MAX;
+		} else {
+			global_log->fatal() << "Wrong deviationReductionOperation given: " << _deviationReductionOperation
+								<< ". Should be 'max' or 'sum'." << std::endl;
+			Simulation::exit(45681);
+		}
+	}
+	global_log->info() << "KDDecomposition uses " << deviationReductionOperation
+					   << " to reduce the deviation within the decompose step." << endl;
```

Comment:
```suggestion
   	{
		std::string deviationReductionOperation;
		xmlconfig.getNodeValue("deviationReductionOperation", deviationReductionOperation);
		if (not deviationReductionOperation.empty()) {
			if (deviationReductionOperation == "sum") {
				_deviationReductionOperation = MPI_SUM;
			} else if (deviationReductionOperation == "max") {
				_deviationReductionOperation = MPI_MAX;
			} else {
				global_log->fatal() << "Wrong deviationReductionOperation given: " << _deviationReductionOperation
									<< ". Should be 'max' or 'sum'." << std::endl;
				Simulation::exit(45681);
			}
		}
		global_log->info() << "KDDecomposition uses " << deviationReductionOperation
					   << " to reduce the deviation within the decompose step." << endl;
	}				   
```

Scope this to avoid confusing variable names.